src/js/languages/default.json

```diff
@@ -72,17 +72,21 @@
     "label": "View"
   },
   "fileInput": {
+    "alert": {
+      "maxSize": "Attention, this file {fileName} is too large. Attach files which are no larger than {maxSize} to continue.",
+      "maxFile": "Attention, attach a maximum of {maxFile} files only. File {fileName} is beyond this limit."
+    },
     "browse": "browse",
     "dropPrompt": "Drop file here or",
     "dropPromptMultiple": "Drop files here or",
     "files": "files",
-    "maxFile": "Attach a maximum of {max} files only.",
+    "maxFile": "Attach a maximum of {maxFile} files only.",
```

Comment:
I think renaming the "max" variable in this message would be a breaking change for anyone relying on it in custom messages. Can you update so it's still called "max"?